config/src/main/java/com/typesafe/config/ConfigRenderOptions.java

```diff
@@ -174,9 +204,53 @@ public String toString() {
             sb.append("formatted,");
         if (json)
             sb.append("json,");
+        if (comparator != null)
```

Comment:
is comparator actually allowed to be null? It looks to me like it isn't (it has to be DefaultComparator if not set explicitly)

Reply:
Changed